build.sh

```diff
@@ -113,6 +117,12 @@ DOCKER_BUILD_ARGS=(
   --build-arg "DOCKER_ORG=${DOCKER_ORG}"
   --build-arg "DOCKER_REPO=${DOCKER_REPO}"
 )
+if [ -n "HTTP_PROXY" ]; then
+  DOCKER_BUILD_ARGS+=( --build-arg "http_proxy=${HTTP_PROXY}" )
```

Comment:
For the sake of consistence, could you try to use the upper-case equivalent of `http_proxy`, i.e. `HTTP_PROXY`? This should be no problem per my interpretation of the [Predefined Build Args](https://docs.docker.com/engine/reference/builder/#predefined-args) documentation.

Reply:
So it turns out only the lowercase forms of these environment variables are taken into account by both wget and pip.
I kept the variable names uppercase when calling build.sh to match the other variables being read by build.sh however I set the lowercase ones with --build-arg so they get properly used inside the docker build.